src/dialogue.h

```diff
@@ -2,318 +2,341 @@
 #ifndef DIALOGUE_H
 #define DIALOGUE_H
 
+#include "npc.h"
+#include "npc_class.h"
+#include "output.h"
+#include "game.h"
+#include "map.h"
+#include "dialogue.h"
```

Comment:
I'm sure this hits a recursion breaker, but...

Reply:
Oops. Fixed.